src/script/util/TimeUtil.ts

```diff
@@ -189,6 +189,17 @@ const mapUnits = (duration: number, rounded: boolean): DiscreteTimeUnit[] => {
 export const formatDuration = (duration: number): DurationUnit => {
   const mappedUnits = mapUnits(duration, true);
   const firstNonZeroUnit = mappedUnits.find(unit => unit.value > 0);
+
+  if (!firstNonZeroUnit) {
+    const seconds = durationUnits().pop();
+
+    return {
+      symbol: seconds.symbol,
+      text: `0 ${seconds.plural}`,
+      value: 0,
+    };
+  }
+
```

Comment:
If `duration` was less then 1000 (that means, less than a second long), this function would throw an error.